src/main/java/org/jabref/model/study/QueryResult.java

```diff
@@ -0,0 +1,24 @@
+package org.jabref.model.study;
+
+import java.util.List;
+
+/**
+ * Represents the result of fetching the results from all active fetchers for a specific query.
+ */
+public class QueryResult {
```

Comment:
Convert to a Java 14 record?

https://dzone.com/articles/a-first-look-at-records-in-java-14